DFC.ServiceTaxonomy.UnitTests/Neo4j/Services/NeoDriverBuilderTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using DFC.ServiceTaxonomy.Neo4j.Configuration;
+using DFC.ServiceTaxonomy.Neo4j.Services;
+using FakeItEasy;
+using Microsoft.Extensions.Options;
+using Xunit;
+using System.Linq;
+using Neo4j.Driver;
+
+namespace DFC.ServiceTaxonomy.UnitTests.Neo4j.Services
+{
+    public class NeoDriverBuilderTests
+    {
+        private readonly IOptionsMonitor<Neo4jConfiguration> _optionsMonitor;
+
+        public NeoDriverBuilderTests()
+        {
+            _optionsMonitor = A.Fake<IOptionsMonitor<Neo4jConfiguration>>();
+            var neo4jConfiguration = A.Fake<Neo4jConfiguration>();
+            neo4jConfiguration.Endpoints = new System.Collections.Generic.List<EndpointConfiguration> { new EndpointConfiguration { Uri = "bolt://example.com", Enabled = true, Primary = true }, new EndpointConfiguration { Uri = "bolt://anotherexample.com", Enabled = true, Primary = false } };
```

Comment:
Is there a `List` in another namespace? If not, could we remove the namespace prefix?
Also, could we split the line up a bit?